src/client/tile.h

```diff
@@ -159,7 +159,10 @@ enum MaterialType{
 	TILE_MATERIAL_LIQUID_TRANSPARENT,
 	TILE_MATERIAL_LIQUID_OPAQUE,
 	TILE_MATERIAL_WAVING_LEAVES,
-	TILE_MATERIAL_WAVING_PLANTS
+	TILE_MATERIAL_WAVING_PLANTS,
+	TILE_MATERIAL_WAVING_WATER_BASIC,
+	TILE_MATERIAL_WAVING_WATER_TRANSPARENT,
+	TILE_MATERIAL_WAVING_WATER_OPAQUE
```

Comment:
Missing trailing comma (yes it is allowed, and actually preferable for neater future diffs)